daemon/api_test.go

```diff
@@ -1183,6 +1186,94 @@ func (s *apiSuite) TestSysInfoLegacyRefresh(c *check.C) {
 	c.Check(rsp.Result, check.DeepEquals, expected)
 }
 
+func (s *apiSuite) testSysInfoSystemMode(c *check.C, mode string) {
+	c.Assert(mode != "", check.Equals, true, check.Commentf("mode is unset for the test"))
+	rec := httptest.NewRecorder()
+
+	// reload dirs for release info to have effect
+	dirs.SetRootDir(dirs.GlobalRootDir)
```

Comment:
could you adjust this comment to explain that this doesn't change anything, and just re-calls the callbacks for the root dir